packages/admin-next/dashboard/src/i18n/translations/en.json

```diff
@@ -1111,12 +1148,12 @@
       "details": "Promotion Details"
     },
     "tabs": {
-      "template": "Template",
-      "promotion": "Promotion",
+      "template": "Type",
+      "promotion": "Details",
       "campaign": "Campaign"
     },
     "fields": {
-      "method": "Method",
+      "method": "Campaign",
```

Comment:
Update key as well

Reply:
Found an exisiting `campaign` key so removed this one instead